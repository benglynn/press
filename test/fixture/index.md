---
template: 'home.pug'
tags:
 - global
---

# Beverage vessels

Welcome!