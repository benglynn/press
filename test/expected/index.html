<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Home</title><script src="/app.js"></script></head><body><header></header><main class="main"><div class="html"><h1>Beverage vessels</h1>
<p>Welcome!</p></div><ul><li><a href="/french-press/">French press</a></li><li><a href="/tea-pot/">Tea pot</a></li></ul></main></body></html>